lms/envs/common.py

```diff
@@ -5541,3 +5541,6 @@ def _should_send_learning_badge_events(settings):
 # .. for now it wil impact country listing in auth flow and user profile.
 # .. eg ['US', 'CA']
 DISABLED_COUNTRIES = []
+
+
+LMS_COMM_DEFAULT_FROM_EMAIL = "no-reply@example.com"
```

Comment:
Will this email be specifically for goal reminder?

Reply:
For all lms communications sent via SES. This is just goal reminder for now but if we move any emails in the future the same address will be used.